Annoying warning with gcc 4.6

We have recently (see the patches listed under #898) acquired a bunch of move constructors and operators that look like this:
```
#ifdef DEAL_II_WITH_CXX11
  /**
   * Move constructor. Creates a new vector by stealing the internal data of
   * the vector @p v.
   *
   * @note This constructor is only available if deal.II is configured with
   * C++11 support.
   */
  BlockVector (BlockVector<Number> &&v) = default;
#endif
```
Unfortunately, with gcc 4.6.3, I get the following, really annoying warning:
```
/.../dealii/include/deal.II/lac/block_vector.h:115:3: warning: unused parameter ‘v’ [-Wunused-parameter]
```

I get a couple of pages of these. What to do with this? I am reluctant to just comment out the name, because we use the name in the documentation. On the other hand, the warnings completely obscure everything else. Suggestions?

--- a/include/deal.II/lac/block_vector.h
+++ b/include/deal.II/lac/block_vector.h
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 1999 - 2015 by the deal.II authors
+// Copyright (C) 1999 - 2016 by the deal.II authors
 //
 // This file is part of the deal.II library.
 //
@@ -107,12 +107,12 @@ public:
 #ifdef DEAL_II_WITH_CXX11
   /**
    * Move constructor. Creates a new vector by stealing the internal data of
-   * the vector @p v.
+   * the given argument vector.
    *
    * @note This constructor is only available if deal.II is configured with
    * C++11 support.
    */
-  BlockVector (BlockVector<Number> &&v) = default;
+  BlockVector (BlockVector<Number> &&/*v*/) = default;
 #endif
 
 
@@ -202,13 +202,13 @@ public:
 
 #ifdef DEAL_II_WITH_CXX11
   /**
-   * Move the given vector. This operator replaces the present vector with @p
-   * v by efficiently swapping the internal data structures.
+   * Move the given vector. This operator replaces the present vector with
+   * the contents of the given argument vector.
    *
    * @note This operator is only available if deal.II is configured with C++11
    * support.
    */
-  BlockVector<Number> &operator= (BlockVector<Number> &&v) = default;
+  BlockVector<Number> &operator= (BlockVector<Number> &&/*v*/) = default;
 #endif
 
   /**

--- a/include/deal.II/lac/block_vector_base.h
+++ b/include/deal.II/lac/block_vector_base.h
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2004 - 2015 by the deal.II authors
+// Copyright (C) 2004 - 2016 by the deal.II authors
 //
 // This file is part of the deal.II library.
 //
@@ -584,14 +584,14 @@ public:
   BlockVectorBase (const BlockVectorBase &V) = default;
 
   /**
-   * Move constructor. Each block of the vector @p V is moved into the current
+   * Move constructor. Each block of the argument vector is moved into the current
    * object if the underlying <code>VectorType</code> is move-constructible,
    * otherwise they are copied.
    *
    * @note This constructor is only available if deal.II is configured with
    * C++11 support.
    */
-  BlockVectorBase (BlockVectorBase &&V) = default;
+  BlockVectorBase (BlockVectorBase &&/*V*/) = default;
 #endif
 
   /**
@@ -742,10 +742,11 @@ public:
 
 #ifdef DEAL_II_WITH_CXX11
   /**
-   * Move assignment operator. Move each block of the vector @p V into the
-   * current object if `VectorType` is move-constructible, otherwise copy them.
+   * Move assignment operator. Move each block of the given argument
+   * vector into the current object if `VectorType` is
+   * move-constructible, otherwise copy them.
    */
-  BlockVectorBase &operator= (BlockVectorBase &&V) = default;
+  BlockVectorBase &operator= (BlockVectorBase &&/*V*/) = default;
 #endif
 
   /**

